cmd/snap/main.go

```diff
@@ -53,9 +55,6 @@ func init() {
 		// in tests or when debugging, enforce the "tidy" lint checks
 		noticef = logger.Panicf
 	}
-
-	// plug/slot sanitization not used nor possible from snap command, make it no-op
-	snap.SanitizePlugsSlots = func(snapInfo *snap.Info) {}
```

Comment:
as per @stolowski considerations we might want to keep doing this at least in the "snap run" case